### ISSUE TITLE
subscriptions plugin

### ISSUE THREAD

**Issue by 7veces7** [WWBN/AVideo#2994]
Hi.
The subscriptions plugin does not work well. Does not allow payments and does not allow deleting and creating plans.
Thank you.
A greeting.

**Comment by DanielnetoDotCom** [comment]
It suppose to.

Can you please provide details?

**Comment by 7veces7** [comment]
Attached screenshot
I can't give you much more details.​
Instructions to pay and delete plans do not work.
________________________________
De: Daniel Neto <notifications@github.com>
Enviado: domingo, 3 de mayo de 2020 17:55
Para: WWBN/AVideo <AVideo@noreply.github.com>
Cc: 7veces7 <sietevecessiete@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [WWBN/AVideo] subscriptions plugin (#2994)


It suppose to.

Can you please provide details?

**Comment by DanielnetoDotCom** [comment]
The screenshots does not come 

Do you mean delete plans from the AVideo platform?

Are you using PayPal or stripe?

Check your log files for some clue .

**Comment by 7veces7** [comment]
Uso Paypal.
________________________________
De: Daniel Neto <notifications@github.com>
Enviado: lunes, 4 de mayo de 2020 11:37
Para: WWBN/AVideo <AVideo@noreply.github.com>
Cc: 7veces7 <sietevecessiete@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [WWBN/AVideo] subscriptions plugin (#2994)


The screenshots does not come

Do you mean delete plans from the AVideo platform?

Are you using PayPal or stripe?

Check your log files for some clue .

**Comment by DanielnetoDotCom** [comment]
Please provide screenshots and log files

**Comment by 7veces7** [comment]
I certainly express myself better in Spanish. Thanks.
--------------------------------------------------------
Te enviado de nuevo una captura de pantalla. 3ª


Te adjunto zip

Te lo envío a la otra cuenta de email también.

________________________________
De: Daniel Neto <notifications@github.com>
Enviado: lunes, 4 de mayo de 2020 15:49
Para: WWBN/AVideo <AVideo@noreply.github.com>
Cc: 7veces7 <sietevecessiete@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [WWBN/AVideo] subscriptions plugin (#2994)


Please provide screenshots and log files

**Comment by DanielnetoDotCom** [comment]
please try to not answer this issue using the email. open the GitHub page and provide screenshots and logs.

also, I receive the zip file with the images and I guess you can not delete the plans that already have subscribers. this is expected. once someone is subscribed you cannot delete. only inactivate.

Also in your logs, I saw som many SQL queries, I have no idea where does it come from. did you change something in the main code?

**Comment by 7veces7** [comment]
**NO permite borrar ninguna suscripción. Aunque no haya nadie suscrito. Este o no activo.**
Por favor compruebalo, si necesitas te doy claves de acceso como administrador y lo miras dentro.
No he modificado el código.
--------------------
** It does NOT allow deleting any subscription. Although no one is subscribed. It is active or not. **
Please check it, if you need the access codes as administrator and you look inside.
He did not modify the code.

-------------------------------------------------------------------------------

![marcados todosCaptura](https://user-images.githubusercontent.com/62806015/81044393-8aabe500-8eb4-11ea-9c9c-475a3d603878.JPG)
![plan](https://user-images.githubusercontent.com/62806015/81044400-8d0e3f00-8eb4-11ea-9fdd-d38b4d14d9b7.JPG)
----------------------------------------------------------------------------------
[1.txt](https://github.com/WWBN/AVideo/files/4579439/1.txt)
[2.txt](https://github.com/WWBN/AVideo/files/4579440/2.txt)
[3.txt](https://github.com/WWBN/AVideo/files/4579441/3.txt)

**Comment by DanielnetoDotCom** [comment]
The files you provided are not the log files. please check here how to get the log files

https://github.com/WWBN/AVideo/wiki/How-to-find-errors-on-AVideo-Platform

![image](https://user-images.githubusercontent.com/18009293/81066400-6a2f5b00-8eb3-11ea-9d1f-b75af099d489.png)

**Comment by DanielnetoDotCom** [comment]
I just notice your site was installed by me.

Just to make sure, can you please update it? https://github.com/WWBN/AVideo/wiki/How-to-Update-your-AVideo-Platform

**Comment by 7veces7** [comment]
Yes, the installation was done by you.
I paid for it, for the program and also for the plugins.
I am not an expert in this and so I prefer to trust professionals and think that things work well.
I have spent many hours trying to solve problems and I don't know how to do some of the things you ask of me. I set up a website like this, but I'm not a programmer and I won't fix errors.
Sorry, I am just a normal user of your products.
-------------------------------------------------- ---------------------------------------
The web already has many things configured, (many hours of work) if necessary, a backup would have to be done first of all.
Please, I ask you to do what is necessary.

**En esta dirección
Inmotube.com + /your/path/to/AVideo/video/avideo.log ------- =   da error 404.**


Thank you

**Comment by DanielnetoDotCom** [comment]
Hi, email to developer@youphptube.com your site admin credentials.

I will take a look into your logs why it is not deleting.

**Comment by DanielnetoDotCom** [comment]
that was your problem

![image](https://user-images.githubusercontent.com/18009293/81090309-e5a10480-8ed3-11ea-9b79-cb1dd3cac9dc.png)

I update it for you, try now.

**Comment by 7veces7** [comment]
Well it seems to work and surely it affected some other things too, don't you think?
Thank you

________________________________
De: Daniel Neto <notifications@github.com>
Enviado: martes, 5 de mayo de 2020 18:27
Para: WWBN/AVideo <AVideo@noreply.github.com>
Cc: 7veces7 <sietevecessiete@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [WWBN/AVideo] subscriptions plugin (#2994)


that was your problem

[image]<https://user-images.githubusercontent.com/18009293/81090309-e5a10480-8ed3-11ea-9b79-cb1dd3cac9dc.png>

I update it for you, try now.

**Comment by DanielnetoDotCom** [comment]
Correct , you had different plugins versions not compatible with the streamer version you had. You all updated now

**Comment by 7veces7** [comment]
Ok.

Why does that appear in many videos?​

The media could not be loaded, either because the server or network failed or because the format is not supported.
________________________________
De: JOSE D <sietevecessiete@hotmail.com>
Enviado: martes, 5 de mayo de 2020 19:17
Para: WWBN/AVideo <reply@reply.github.com>
Asunto: RE: [WWBN/AVideo] subscriptions plugin (#2994)

Well it seems to work and surely it affected some other things too, don't you think?
Thank you

________________________________
De: Daniel Neto <notifications@github.com>
Enviado: martes, 5 de mayo de 2020 18:27
Para: WWBN/AVideo <AVideo@noreply.github.com>
Cc: 7veces7 <sietevecessiete@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [WWBN/AVideo] subscriptions plugin (#2994)


that was your problem

[image]<https://user-images.githubusercontent.com/18009293/81090309-e5a10480-8ed3-11ea-9b79-cb1dd3cac9dc.png>

I update it for you, try now.

**Comment by DanielnetoDotCom** [comment]
Please open another issue regarding "The media could not be loaded"
